recipes/kcov/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+import os
+from shutil import rmtree, move
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class KcovConan(ConanFile):
+    name = "kcov"
+    license = "GPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index/"
+    homepage = "http://simonkagstrom.github.io/kcov/index.html"
+    description = "Code coverage tool for compiled programs, Python and Bash\
+        which uses debugging information to collect and report data without\
+            special compilation options"
+    topics = ("coverage", "linux", "debug")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    requires = ["zlib/1.2.11",
+                "libiberty/9.1.0",
+                "libcurl/7.64.1"]
+    generators = "cmake"
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration(
+                  "kcov can not be built by Visual Studio.")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def system_requirements(self):
```

Comment:
System requirements should not be used in CCI.

Reply:
@uilianries should i use it like backward-cpp does? warn if packages are note installed?
https://github.com/conan-io/conan-center-index/blob/37dda8777ea3a4cb15467f78acb9f8799d8e1092/recipes/backward-cpp/all/conanfile.py#L55-L81

Reply:
Yes, it will be much better